pkg/options/options.go

```diff
@@ -59,7 +61,9 @@ func NewOptions() *Options {
 func (o *Options) AddFlags() {
 	o.flags = pflag.NewFlagSet("", pflag.ExitOnError)
 	// add klog flags
-	o.flags.AddGoFlagSet(flag.CommandLine)
+	klogFlags := flag.NewFlagSet("klog", flag.ExitOnError)
```

Comment:
This may need reviewing. After taking in the new k8s deps, this seemed to be the only way to get this code to work.

For some reason, the flagSet never included the `logtostderr` flag with the current code after the dependency update.

Reply:
@mxinden @brancz  Please review this. I did a bit of code digging and this was how I got it to work. It throws a panic with the new depped libraries otherwise. 

If something about this does not seem okay, I'll try to find some time to take a deeper look. (I am in between switching employers, so I will take a while. Apologies for that)

Reply:
This looks good to me. I understand this way it doesn't panic anymore, right?

Reply:
Yes that is correct